### DISCUSSION TITLE
DateTime in UTC

### DISCUSSION THREAD

**Discussion by yetibrain** [nunit/nunit#3996] [Q&A] [answered]
Hi All,

is there a way to run NUnit so that NUnit will convert the local time to UTC time when creating the NUnit result file?

Thanks for any hint.

Kind regards,

yb

**Comment by rprouse** [comment, marked as answer]
NUnit test results do use UTC, `start-time="2021-11-23T03:43:45.7213186Z"`

Are you talking about results like test failure methods? If so, there is no way to do that. We output what we are given.

**Reply by yetibrain** [reply]
Yes, i mean the start-time property, didn't see the UTC format, guess it's a NUnit 3.0 example, right?

**Reply by rprouse** [reply]
@yetibrain if you are asking about NUnit 2.x format, it is in local and AFAIK there is no way to change that.
